# Final Project of Sampling Method

### A.	Introduction & Background

###Isu Pengangguran dan Upaya Pemerintah Provinsi DKI Jakarta dalam Penurunan Tingkat Pengangguran Terbuka (TPT)
Pengangguran merupakan salah satu isu krusial yang mendapatkan perhatian serius dari pemerintah, baik di tingkat pusat maupun daerah. Tingginya tingkat pengangguran dapat berdampak negatif terhadap kesejahteraan masyarakat, meningkatkan angka kemiskinan, serta memicu berbagai masalah sosial yang dapat mengganggu stabilitas dan keamanan wilayah. Oleh karena itu, pengelolaan dan pengurangan tingkat pengangguran menjadi salah satu prioritas utama dalam perencanaan dan pelaksanaan kebijakan ekonomi dan sosial.

###Tingkat Pengangguran Terbuka (TPT) dan Target Pemerintah
Pemerintah Provinsi DKI Jakarta, sebagai salah satu daerah dengan kepadatan penduduk dan kegiatan ekonomi yang tinggi, menargetkan penurunan Tingkat Pengangguran Terbuka (TPT) secara bertahap dari 7,14% pada tahun 2017 menjadi 5,00% pada tahun 2022. Target ini mencerminkan komitmen pemerintah dalam menciptakan lapangan pekerjaan yang lebih banyak dan berkualitas serta mengurangi ketidakstabilan ekonomi yang diakibatkan oleh tingginya pengangguran.

###Tantangan yang Dihadapi
Namun, pencapaian target tersebut tidaklah mudah dan menghadapi berbagai tantangan besar, terutama dengan adanya pandemi Covid-19 yang merebak pada awal tahun 2020. Pandemi ini telah menyebabkan disrupsi besar dalam berbagai sektor ekonomi, mempengaruhi ketersediaan lapangan kerja, serta mengubah dinamika pasar tenaga kerja. Banyak perusahaan yang mengalami penurunan pendapatan, melakukan pemutusan hubungan kerja (PHK), dan menunda perekrutan, yang pada akhirnya berdampak pada peningkatan jumlah pengangguran.

###Karakteristik Pengangguran
Pengangguran tidak hanya dipengaruhi oleh kondisi makroekonomi, tetapi juga memiliki berbagai karakteristik yang bervariasi. Beberapa karakteristik penting dari pengangguran yang perlu dikaji lebih lanjut meliputi:
Pendidikan: Tingkat pendidikan juga mempengaruhi tingkat pengangguran. Individu dengan tingkat pendidikan yang lebih rendah seringkali mengalami pengangguran yang lebih tinggi dibandingkan dengan mereka yang memiliki pendidikan tinggi. Hal ini berkaitan dengan keterampilan yang dimiliki dan kesesuaian dengan kebutuhan pasar kerja.


Berdasarkan hal tersebut, maka kami membuat rancangan sampling mengenai tingkat pengangguran di DKI Jakarta untuk menghadapi kondisi ekonomi mendatang. Harapan yang didapat dari penelitian ini nantinya diperoleh jumlah yang tepat berapa banyak penagguran agar nantinya dapat dibina untuk mendapatkan pekerjaan nantinya sesuai dengan starata pendidikan

### B.	Import Package

In [ ]:
# Load data manipulation package
import numpy as np
import pandas as pd
import math
import copy

# Load data visualization package
import matplotlib.pyplot as plt
import seaborn as sns

# Load statistic package
from scipy import stats

### C. Import Dataset

In [ ]:
# Load dataset
prior_2021_rev = pd.read_excel('1.Revenue.xlsx')
#prior_2021_percent = pd.read_csv('2021_Informal.csv')
prior_2021_tingkatan = pd.read_excel('3.Tingkat Pendidikan.xlsx')
prior_2021_kota = pd.read_excel('2.Tingkat Kota.xlsx')
prior_2021_kec = pd.read_csv('kecamatan.csv')

In [ ]:
prior_2021_tingkatan

Strata Pendidikan  Persentase
0               SMP       13.39
1       SD Ke bawah       10.09
2      DIV/S1/S2/S3       10.50
3         DI/II/III        4.53
4               SMK       34.41
5               SMA       27.08

In [ ]:
prior_2021_kota

Kota  Persentase
0  Kepulauan Seribu        0.14
1   Jakarta Selatan       21.37
2     Jakarta Timur       22.48
3     Jakarta Pusat        8.81
4    Jakartar Barat       28.03
5     Jakarta Utara       19.18

In [ ]:
prior_2021_kec

Kabupaten/Kota  Jumlah Kecamatan
0  Kepulauan Seribu                 2
1   Jakarta Selatan                10
2     Jakarta Timur                10
3     Jakarta Pusat                 8
4     Jakarta Barat                 8
5     Jakarta Utara                 6

In [ ]:
prior_2021_rev

Kota  Rata-rata
0  Kepulauan Seribu        794
1   Jakarta Selatan     122390
2     Jakarta Timur     128739
3     Jakarta Pusat      50444
4    Jakartar Barat     160554
5     Jakarta Utara     109859
6       DKI Jakarta     572780

In [ ]:
#Cleasing dataset untuk kolom "Rata-rata"
rev_kota_2021 = prior_2021_rev
rev_kota_2021["Rata-rata"] = rev_kota_2021["Rata-rata"].astype(float)
rev_kota_2021

Kota  Rata-rata
0  Kepulauan Seribu      794.0
1   Jakarta Selatan   122390.0
2     Jakarta Timur   128739.0
3     Jakarta Pusat    50444.0
4    Jakartar Barat   160554.0
5     Jakarta Utara   109859.0
6       DKI Jakarta   572780.0

### D. Sampling Plan

* Populasi

Populasi dalam penelitian ini terdiri dari beberapa tahapan, yaitu sebagai berikut
1. Populasi Tahap 1 : seluruh Kota di Jakarta
2. Populasi Tahap 2: Seluruh Kecamatan di Kabupaten/Kota di DKI Jakarta
3. Populasi tahap 3 : Seluruh PENGANGGURAN di kecamatan DKI Jakarta


* Populasi Sasaran

Populasi sasaran adalah seluruh penduduk yang merupakan Usia layak Kerja di DKI Jakarta

* Kerangka Pengambilan Sampel

Untuk memperoleh informasi dari populasi, penulis menggunakan data dari https://jakarta.bps.go.id/id/publication/2021/10/28/445d7e1cdf4802df0fbdd8b8/profil-pengangguran-provinsi-dki-jakarta-2020.html.

* Unit sampel

Untuk penelitian yang terkait dengan tingkat Pengangguran, Sampel diambil berdasarkan unit-unit berikut:
Tahap Pertama : Seluruh Kabupaten/Kota di DKI jakarta
Tahap Kedua : seluruh kecamatan di kabupaten/kota DKI jakarta
Tahap Ketiga : seluruh pengangguran yang layak kerja di DKI Jakarta


* Unit observasi

Dalam hal ini, unit observasi adalah Orang yang layak kerja di DKI jakarta karena objeknya adalah jumlah pengganguran


* Unit analisis

Karena ingin melihat tingkat penagguran di DKI jakarta, maka unit analisis yang digunakan adalah pada level usia layak kerja berdasarkan tingkat pendidikan

* Karakteristik yang diteliti

Karena karakteristik yang diteliti adalah tingkat pengangguran di DKI jakarta. maka bidang minat utama adalah jumlah pengganguran berdasarkan tingkat pendidikan

* Nilai karakteristik yang diestimasi

Nilai karakteristik yang diestimasi adalah Berapa banyak pengangguran berdasarkan Jenis Kelamin dan pendidikan



* Metode pengambilan sampel

Metode pengambilan sampel yang digunakan adalah gabungan dari stratifikasi, cluster sampling dan probabilitas proporsional terhadap ukuran atau biasa dikenal dengan multistage sampling karena untuk mengakomodir strata tingkat pendapatan hingga jenjang pendidikan diperlukan stratified sampling dan kabupaten/kota, sedangkan cluster sampling dilakukan untuk memilih klaster kecamatan mana saja yang akan dijadikan sampel. Selanjutnya pengambilan sampel dilakukan dengan metode SRS.

### E.  Estimating the Needs of the Number of Samples

*Ada* 4 tahap dalam pengambilan sampel ini **(Pengambilan Sampel Multitahap)**

a. Tahap 1: Membagi populasi menjadi 4 strata (Stratified Sampling Tahap 1)

b. Tahap 2: Membagi populasi menjadi 6 strata (Stratified Sampling Tahap 2)

c. Tahap 3: Cluster sampling pada tahap 1 di setiap strata (Distrik)

d. Tahap 4: Sampling dengan SRS

#### Stage 1: Dividing the population into 4 strata (Stratified Sampling Stage 1)

Pembagian ini dilakukan karena asumsi awal bahwa terdapat perbedaan pengangguran berdasarkan jenjang pendidikan. Maka penduduk dibagi menjadi SMP/SD Ke bawah/ DIV,S1,S2,S3/DI,II,III/SMK/SMA. Persentase rata-rata diperoleh dari data BPS DKI Jakarta.

Total Cost Formula :

$$
C = c_{0} + c_{1}n + c_{2}nm
$$

In [ ]:
# Assumption Formula
c0 = 4_000_000
c1 = 50_000
c2 = 50_000
N = 5
n_ = 5
M = 1000
nm = 1000


$$
m_{\text{opt}} = \sqrt{
\cfrac{c_{1} \sigma_{w}^{2}}
{c_{2} \left( \sigma_{b}^{2} - \cfrac{\sigma_{w}^{2}}{\bar{M}} \right)}
}
$$

$$
\sigma_{b}^{2} = \cfrac{1}{N-1} \sum_{i=1}^{N} (\mu_{i} - \mu)^{2}
$$

$$
\sigma_{w}^{2} = \cfrac{1}{N} \sum_{i=1}^{N} \sigma_{i}^{2}
$$

In [ ]:
# Average Revenue
rev_dki_2021 = rev_kota_2021.loc[6::,["Rata-rata"]]
rev_dki_2021 = rev_dki_2021["Rata-rata"].values[0]

In [ ]:
rev_kota_2021["Variance"] = (rev_kota_2021["Rata-rata"]-rev_dki_2021)**2

In [ ]:
rev_kota_2021

Kota  Rata-rata      Variance
0  Kepulauan Seribu      794.0  3.271680e+11
1   Jakarta Selatan   122390.0  2.028512e+11
2     Jakarta Timur   128739.0  1.971724e+11
3     Jakarta Pusat    50444.0  2.728349e+11
4    Jakartar Barat   160554.0  1.699303e+11
5     Jakarta Utara   109859.0  2.142959e+11
6       DKI Jakarta   572780.0  0.000000e+00

Kalkulasi Performaed untuk mencari find σ_b^2 dan σ_w^2

Karena kita sudah menghitung varians data, kita sekarang dapat menghitung σ_b^2 dan σ_w^2 berdasarkan rumus di atas. Akhirnya, kita bisa mendapatkan nilai untuk m.opt

In [ ]:
# Value of sigma_b^2 & sigma_w^2
sigma_b = (1/(N-1))*(rev_kota_2021["Variance"].sum())
sigma_w = (1/(N))*(rev_kota_2021["Variance"].sum())
print("Nilai dai sigma_b^2 adalah", sigma_b)
print("Nilai dai sigma_w^2 adalah", sigma_w)

Nilai dai sigma_b^2 adalah 346063142547.5
Nilai dai sigma_w^2 adalah 276850514038.0


In [ ]:
# Calculate value of m_opt
m_opt = math.sqrt(c1*sigma_w/(c2*(sigma_b-(sigma_w/M))))
print("The value of m_opt is", m_opt)

The value of m_opt is 0.8947851766820504


Langkah selanjutnya adalah menghitung total biaya alokasi (C) dengan 5 PSU, yang sesuai dengan jumlah kota yang dipilih dan mengasumsikan ukuran sampel data sebelumnya sebesar 1000 (data yang diasumsikan). Hal ini dapat dihitung sebagai berikut:

In [ ]:
# Calculate total allocation cost with C formula
C = c0 + c1*n_+c2*nm
print("Total allocation cost is",C)

Total allocation cost is 54250000



Total biaya alokasi telah dihitung, jadi kita dapat menggunakan persamaan berikut untuk menentukan ukuran sampel yang diperlukan.

$$
n = \cfrac{C - c_{0}}{c_{1} + c_{2} m_{\text{opt}}}
$$

In [ ]:
# Calculate total sample or n
n = (C-c0)/(c1+c2*m_opt)
n = math.ceil(n)
print("Total sample needed is", n)

Total sample needed is 531


In [ ]:
# Copy of School Strata
df_2021_strata = copy.copy(prior_2021_tingkatan)

In [ ]:
df_2021_strata

Strata Pendidikan  Persentase
0               SMP       13.39
1       SD Ke bawah       10.09
2      DIV/S1/S2/S3       10.50
3         DI/II/III        4.53
4               SMK       34.41
5               SMA       27.08

In [ ]:
df_2021_strata["Persentase (%)"] = df_2021_strata["Persentase"]/100

In [ ]:
# Dividing the sample based on school strata
df_2021_strata["Total Sample"] = round(df_2021_strata["Persentase (%)"]*n,0)

The total of the sample will be divided into the first strata for the Education level group using *proportional allocation* and the number of samples is obtained as follows:

In [ ]:
df_2021_strata

Strata Pendidikan  Persentase  Persentase (%)  Total Sample
0               SMP       13.39          0.1339          71.0
1       SD Ke bawah       10.09          0.1009          54.0
2      DIV/S1/S2/S3       10.50          0.1050          56.0
3         DI/II/III        4.53          0.0453          24.0
4               SMK       34.41          0.3441         183.0
5               SMA       27.08          0.2708         144.0

#### Stage 2: Dividing the population into 6 strata (Stratified Sampling Stage 2)

Pembagian ini dilakukan karena asumsi awal adanya perbedaan tingkat Penagngguran menurut kabupaten/kota. Maka populasi dibagi menjadi Kepulauan Seribu, Jakarta Selatan, Jakarta Timur, Jakarta Pusat, Jakarta Barat dan Jakarta Utara. Jumlah sampel yang akan digunakan menggunakan alokasi proporsional.

In [ ]:
prior_2021_kota

Kota  Persentase
0  Kepulauan Seribu        0.14
1   Jakarta Selatan       21.37
2     Jakarta Timur       22.48
3     Jakarta Pusat        8.81
4    Jakartar Barat       28.03
5     Jakarta Utara       19.18

In [ ]:
tingkat_pendidikan = df_2021_strata["Strata Pendidikan"].values.tolist()
kota_dki = prior_2021_kota["Kota"].values.tolist()

In [ ]:
tingkat_pendidikan

['SMP', 'SD Ke bawah', 'DIV/S1/S2/S3', 'DI/II/III', 'SMK', 'SMA']

In [ ]:
kota_dki

['Kepulauan Seribu',
 'Jakarta Selatan',
 'Jakarta Timur',
 'Jakarta Pusat',
 'Jakartar Barat',
 'Jakarta Utara']

In [ ]:
# Daftar kategori pendidikan
pendidikan = tingkat_pendidikan

# Daftar lokasi
kota = kota_dki

# Buat DataFrame untuk pendidikan dan kota
df_pendidikan = pd.DataFrame({'Pendidikan': pendidikan})
df_kota = pd.DataFrame({'Kota': kota})

# Lakukan cross join menggunakan merge dengan parameter 'how' = 'cross'
rancang_sampling = pd.merge(df_pendidikan,df_kota, how='cross')


In [ ]:
rancang_sampling

Pendidikan              Kota
0            SMP  Kepulauan Seribu
1            SMP   Jakarta Selatan
2            SMP     Jakarta Timur
3            SMP     Jakarta Pusat
4            SMP    Jakartar Barat
5            SMP     Jakarta Utara
6    SD Ke bawah  Kepulauan Seribu
7    SD Ke bawah   Jakarta Selatan
8    SD Ke bawah     Jakarta Timur
9    SD Ke bawah     Jakarta Pusat
10   SD Ke bawah    Jakartar Barat
11   SD Ke bawah     Jakarta Utara
12  DIV/S1/S2/S3  Kepulauan Seribu
13  DIV/S1/S2/S3   Jakarta Selatan
14  DIV/S1/S2/S3     Jakarta Timur
15  DIV/S1/S2/S3     Jakarta Pusat
16  DIV/S1/S2/S3    Jakartar Barat
17  DIV/S1/S2/S3     Jakarta Utara
18     DI/II/III  Kepulauan Seribu
19     DI/II/III   Jakarta Selatan
20     DI/II/III     Jakarta Timur
21     DI/II/III     Jakarta Pusat
22     DI/II/III    Jakartar Barat
23     DI/II/III     Jakarta Utara
24           SMK  Kepulauan Seribu
25           SMK   Jakarta Selatan
26           SMK     Jakarta Timur
27           SMK     Jakarta Pusat
28           SMK    Jakartar Barat
29           SMK     Jakarta Utara
30           SMA  Kepulauan Seribu
31           SMA   Jakarta Selatan
32           SMA     Jakarta Timur
33           SMA     Jakarta Pusat
34           SMA    Jakartar Barat
35           SMA     Jakarta Utara

disebabkan oleh kecilnya persentase pada kepulauan seribu maka ditetapkan batas minimum 1 agar tidak ada 0 pada total sample

In [ ]:
rancang_sampling["Total Sample"] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][0])
rancang_sampling["Total Sample"] = rancang_sampling["Total Sample"].apply(lambda x: max(x, 1))  # Menetapkan batas minimum 1


In [ ]:
rancang_sampling

Pendidikan              Kota  Total Sample
0            SMP  Kepulauan Seribu           1.0
1            SMP   Jakarta Selatan          15.0
2            SMP     Jakarta Timur          16.0
3            SMP     Jakarta Pusat           6.0
4            SMP    Jakartar Barat          20.0
5            SMP     Jakarta Utara          14.0
6    SD Ke bawah  Kepulauan Seribu           NaN
7    SD Ke bawah   Jakarta Selatan           NaN
8    SD Ke bawah     Jakarta Timur           NaN
9    SD Ke bawah     Jakarta Pusat           NaN
10   SD Ke bawah    Jakartar Barat           NaN
11   SD Ke bawah     Jakarta Utara           NaN
12  DIV/S1/S2/S3  Kepulauan Seribu           NaN
13  DIV/S1/S2/S3   Jakarta Selatan           NaN
14  DIV/S1/S2/S3     Jakarta Timur           NaN
15  DIV/S1/S2/S3     Jakarta Pusat           NaN
16  DIV/S1/S2/S3    Jakartar Barat           NaN
17  DIV/S1/S2/S3     Jakarta Utara           NaN
18     DI/II/III  Kepulauan Seribu           NaN
19     DI/II/III   Jakarta Selatan           NaN
20     DI/II/III     Jakarta Timur           NaN
21     DI/II/III     Jakarta Pusat           NaN
22     DI/II/III    Jakartar Barat           NaN
23     DI/II/III     Jakarta Utara           NaN
24           SMK  Kepulauan Seribu           NaN
25           SMK   Jakarta Selatan           NaN
26           SMK     Jakarta Timur           NaN
27           SMK     Jakarta Pusat           NaN
28           SMK    Jakartar Barat           NaN
29           SMK     Jakarta Utara           NaN
30           SMA  Kepulauan Seribu           NaN
31           SMA   Jakarta Selatan           NaN
32           SMA     Jakarta Timur           NaN
33           SMA     Jakarta Pusat           NaN
34           SMA    Jakartar Barat           NaN
35           SMA     Jakarta Utara           NaN

In [ ]:
rancang_sampling["Total Sample"].iloc[6:12] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][1])
rancang_sampling["Total Sample"] = rancang_sampling["Total Sample"].apply(lambda x: max(x, 1))  # Menetapkan batas minimum 1


<ipython-input-497-2cfeb6781c0a>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Sample"].iloc[6:12] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][1])


In [ ]:
rancang_sampling["Total Sample"].iloc[12:18] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][2])
rancang_sampling["Total Sample"] = rancang_sampling["Total Sample"].apply(lambda x: max(x, 1))  # Menetapkan batas minimum 1


<ipython-input-498-640a5398a3f9>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Sample"].iloc[12:18] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][2])


In [ ]:
rancang_sampling["Total Sample"].iloc[18:24] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][3])
rancang_sampling["Total Sample"] = rancang_sampling["Total Sample"].apply(lambda x: max(x, 1))  # Menetapkan batas minimum 1


<ipython-input-499-c8fd1b139b87>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Sample"].iloc[18:24] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][3])


In [ ]:
rancang_sampling["Total Sample"].iloc[24:30] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][4])
rancang_sampling["Total Sample"] = rancang_sampling["Total Sample"].apply(lambda x: max(x, 1))  # Menetapkan batas minimum 1


<ipython-input-500-705aa179c22a>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Sample"].iloc[24:30] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][4])


In [ ]:
rancang_sampling

Pendidikan              Kota  Total Sample
0            SMP  Kepulauan Seribu           1.0
1            SMP   Jakarta Selatan          15.0
2            SMP     Jakarta Timur          16.0
3            SMP     Jakarta Pusat           6.0
4            SMP    Jakartar Barat          20.0
5            SMP     Jakarta Utara          14.0
6    SD Ke bawah  Kepulauan Seribu           1.0
7    SD Ke bawah   Jakarta Selatan          12.0
8    SD Ke bawah     Jakarta Timur          12.0
9    SD Ke bawah     Jakarta Pusat           5.0
10   SD Ke bawah    Jakartar Barat          15.0
11   SD Ke bawah     Jakarta Utara          10.0
12  DIV/S1/S2/S3  Kepulauan Seribu           1.0
13  DIV/S1/S2/S3   Jakarta Selatan          12.0
14  DIV/S1/S2/S3     Jakarta Timur          13.0
15  DIV/S1/S2/S3     Jakarta Pusat           5.0
16  DIV/S1/S2/S3    Jakartar Barat          16.0
17  DIV/S1/S2/S3     Jakarta Utara          11.0
18     DI/II/III  Kepulauan Seribu           1.0
19     DI/II/III   Jakarta Selatan           5.0
20     DI/II/III     Jakarta Timur           5.0
21     DI/II/III     Jakarta Pusat           2.0
22     DI/II/III    Jakartar Barat           7.0
23     DI/II/III     Jakarta Utara           5.0
24           SMK  Kepulauan Seribu           1.0
25           SMK   Jakarta Selatan          39.0
26           SMK     Jakarta Timur          41.0
27           SMK     Jakarta Pusat          16.0
28           SMK    Jakartar Barat          51.0
29           SMK     Jakarta Utara          35.0
30           SMA  Kepulauan Seribu           NaN
31           SMA   Jakarta Selatan           NaN
32           SMA     Jakarta Timur           NaN
33           SMA     Jakarta Pusat           NaN
34           SMA    Jakartar Barat           NaN
35           SMA     Jakarta Utara           NaN

In [ ]:
rancang_sampling["Total Sample"].iloc[30:36] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][5])
rancang_sampling["Total Sample"] = rancang_sampling["Total Sample"].apply(lambda x: max(x, 1))  # Menetapkan batas minimum 1


<ipython-input-502-444f19e0bc0d>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Sample"].iloc[30:36] = round((prior_2021_kota["Persentase"].iloc[0:6]/100)*df_2021_strata["Total Sample"][5])


In [ ]:
rancang_sampling

Pendidikan              Kota  Total Sample
0            SMP  Kepulauan Seribu           1.0
1            SMP   Jakarta Selatan          15.0
2            SMP     Jakarta Timur          16.0
3            SMP     Jakarta Pusat           6.0
4            SMP    Jakartar Barat          20.0
5            SMP     Jakarta Utara          14.0
6    SD Ke bawah  Kepulauan Seribu           1.0
7    SD Ke bawah   Jakarta Selatan          12.0
8    SD Ke bawah     Jakarta Timur          12.0
9    SD Ke bawah     Jakarta Pusat           5.0
10   SD Ke bawah    Jakartar Barat          15.0
11   SD Ke bawah     Jakarta Utara          10.0
12  DIV/S1/S2/S3  Kepulauan Seribu           1.0
13  DIV/S1/S2/S3   Jakarta Selatan          12.0
14  DIV/S1/S2/S3     Jakarta Timur          13.0
15  DIV/S1/S2/S3     Jakarta Pusat           5.0
16  DIV/S1/S2/S3    Jakartar Barat          16.0
17  DIV/S1/S2/S3     Jakarta Utara          11.0
18     DI/II/III  Kepulauan Seribu           1.0
19     DI/II/III   Jakarta Selatan           5.0
20     DI/II/III     Jakarta Timur           5.0
21     DI/II/III     Jakarta Pusat           2.0
22     DI/II/III    Jakartar Barat           7.0
23     DI/II/III     Jakarta Utara           5.0
24           SMK  Kepulauan Seribu           1.0
25           SMK   Jakarta Selatan          39.0
26           SMK     Jakarta Timur          41.0
27           SMK     Jakarta Pusat          16.0
28           SMK    Jakartar Barat          51.0
29           SMK     Jakarta Utara          35.0
30           SMA  Kepulauan Seribu           1.0
31           SMA   Jakarta Selatan          31.0
32           SMA     Jakarta Timur          32.0
33           SMA     Jakarta Pusat          13.0
34           SMA    Jakartar Barat          40.0
35           SMA     Jakarta Utara          28.0

#### Stage 3 : Cluster sampling in stage 1 in each stratum (District)

Distribusi jumlah klaster kecamatan yang dijadikan sampel dihitung dengan menggunakan perhitungan desain sampel klaster.

In [ ]:
# Mengelompokkan data berdasarkan kolom "Kota" dan menjumlahkan nilai pada kolom "Total Sample"
sample_kota = rancang_sampling.groupby("Kota")["Total Sample"].sum()
sample_kota

Kota
Jakarta Pusat        47.0
Jakarta Selatan     114.0
Jakarta Timur       119.0
Jakarta Utara       103.0
Jakartar Barat      149.0
Kepulauan Seribu      6.0
Name: Total Sample, dtype: float64

In [ ]:
# Mengambil nilai total sampel untuk masing-masing kota
M_sample_ks = sample_kota.iloc[5]  # Kepulauan Seribu
M_sample_js = sample_kota.iloc[2]  # Jakarta Selatan
M_sample_jt = sample_kota.iloc[3]  # Jakarta Timur
M_sample_jp = sample_kota.iloc[1]  # Jakarta Pusat
M_sample_jb = sample_kota.iloc[0]  # Jakarta Barat
M_sample_ju = sample_kota.iloc[4]  # Jakarta Utara

# Menggabungkan nilai-nilai tersebut dalam sebuah list
M_sample_kec = [M_sample_ks, M_sample_js, M_sample_jt, M_sample_jp, M_sample_jb, M_sample_ju]
M_sample_kec


[6.0, 119.0, 103.0, 114.0, 47.0, 149.0]

In [ ]:
prior_2021_kec

Kabupaten/Kota  Jumlah Kecamatan
0  Kepulauan Seribu                 2
1   Jakarta Selatan                10
2     Jakarta Timur                10
3     Jakarta Pusat                 8
4     Jakarta Barat                 8
5     Jakarta Utara                 6

In [ ]:
#prior_2021_kec

In [ ]:
#M of each subdistrict
prior_2021_kec["M Kecamatan"] = M_sample_kec

In [ ]:
prior_2021_kec

Kabupaten/Kota  Jumlah Kecamatan  M Kecamatan
0  Kepulauan Seribu                 2          6.0
1   Jakarta Selatan                10        119.0
2     Jakarta Timur                10        103.0
3     Jakarta Pusat                 8        114.0
4     Jakarta Barat                 8         47.0
5     Jakarta Utara                 6        149.0

In [ ]:
#Assumption of B Value
B = 10000000

In [ ]:
rev_kota_2021["y"] =  rev_kota_2021["Rata-rata"][0:6]*prior_2021_kec["M Kecamatan"]

In [ ]:
rev_kota_2021["miu x m"] = rev_dki_2021*prior_2021_kec["M Kecamatan"]

In [ ]:
rev_kota_2021["(y-miu x m)^2"] = rev_kota_2021["y"] - rev_kota_2021["miu x m"]
rev_kota_2021["(y-miu x m)^2"] = rev_kota_2021["(y-miu x m)^2"]**2

In [ ]:
rev_kota_2021["sigma c"] = rev_kota_2021["(y-miu x m)^2"]/(prior_2021_kec["Jumlah Kecamatan"]-1)

In [ ]:
rev_kota_2021["D"] = B**2/(4*prior_2021_kec["Jumlah Kecamatan"]**2)

In [ ]:
rev_kota_2021["n"] = round((prior_2021_kec["Jumlah Kecamatan"]*rev_kota_2021["sigma c"])/(prior_2021_kec["Jumlah Kecamatan"]*rev_kota_2021["D"]+rev_kota_2021["sigma c"]))

In [ ]:
#Adjustment n value of Kepulauan Seribu
rev_kota_2021["y"][6] = 0
rev_kota_2021["miu x m"][6] = 0
rev_kota_2021["sigma c"][6] = 0
rev_kota_2021["D"][6] = 0
rev_kota_2021["n"][6] = 0
rev_kota_2021["n"][0] = 1

<ipython-input-517-3668ec69df1d>:2: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rev_kota_2021["y"][6] = 0
<ipython-input-517-3668ec69df1d>:3: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rev_kota_2021["miu x m"][6] = 0
<ipython-input-517-3668ec69df1d>:4: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rev_kota_2021["sigma c"][6] = 0
<ipython-input-517-3668ec69df1d>:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a

In [ ]:
rev_kota_2021

Kota  Rata-rata      Variance           y     miu x m  \
0  Kepulauan Seribu      794.0  3.271680e+11      4764.0   3436680.0   
1   Jakarta Selatan   122390.0  2.028512e+11  14564410.0  68160820.0   
2     Jakarta Timur   128739.0  1.971724e+11  13260117.0  58996340.0   
3     Jakarta Pusat    50444.0  2.728349e+11   5750616.0  65296920.0   
4    Jakartar Barat   160554.0  1.699303e+11   7546038.0  26920660.0   
5     Jakarta Utara   109859.0  2.142959e+11  16368991.0  85344220.0   
6       DKI Jakarta   572780.0  0.000000e+00         0.0         0.0   

   (y-miu x m)^2       sigma c             D     n  
0   1.177805e+13  1.177805e+13  6.250000e+12   1.0  
1   2.872575e+15  3.191750e+14  2.500000e+11  10.0  
2   2.091802e+15  2.324225e+14  2.500000e+11  10.0  
3   3.545762e+15  5.065375e+14  3.906250e+11   8.0  
4   3.753760e+14  5.362514e+13  3.906250e+11   8.0  
5   4.757582e+15  9.515164e+14  6.944444e+11   6.0  
6            NaN  0.000000e+00  0.000000e+00   0.0

#### Stage 4 : Sampling with SRS

Berikut ini adalah hasil rekapitulasi untuk rancangan sampling dengan pembagian disesuaikan dengan rancangan sampling dan perhitungan diatas

In [ ]:
rancang_sampling["Total Kecamatan yang disampling"] = rev_kota_2021["n"][0:6]

In [ ]:
rancang_sampling["Total Kecamatan yang disampling"][6:12] = rev_kota_2021["n"][0:6]

<ipython-input-520-7d1b174a1904>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Kecamatan yang disampling"][6:12] = rev_kota_2021["n"][0:6]


In [ ]:
rancang_sampling["Total Kecamatan yang disampling"][12:18] = rev_kota_2021["n"][0:6]

<ipython-input-521-ab827815e4ce>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Kecamatan yang disampling"][12:18] = rev_kota_2021["n"][0:6]


In [ ]:
rancang_sampling["Total Kecamatan yang disampling"][18:24] = rev_kota_2021["n"][0:6]

<ipython-input-522-260471911706>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Kecamatan yang disampling"][18:24] = rev_kota_2021["n"][0:6]


In [ ]:
rancang_sampling["Total Kecamatan yang disampling"][24:30] = rev_kota_2021["n"][0:6]

<ipython-input-523-9acbb30f412c>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Kecamatan yang disampling"][24:30] = rev_kota_2021["n"][0:6]


In [ ]:
rancang_sampling["Total Kecamatan yang disampling"][30:36] = rev_kota_2021["n"][0:6]

<ipython-input-524-56177ded105a>:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  rancang_sampling["Total Kecamatan yang disampling"][30:36] = rev_kota_2021["n"][0:6]


In [ ]:
rancang_sampling

Pendidikan              Kota  Total Sample  \
0            SMP  Kepulauan Seribu           1.0   
1            SMP   Jakarta Selatan          15.0   
2            SMP     Jakarta Timur          16.0   
3            SMP     Jakarta Pusat           6.0   
4            SMP    Jakartar Barat          20.0   
5            SMP     Jakarta Utara          14.0   
6    SD Ke bawah  Kepulauan Seribu           1.0   
7    SD Ke bawah   Jakarta Selatan          12.0   
8    SD Ke bawah     Jakarta Timur          12.0   
9    SD Ke bawah     Jakarta Pusat           5.0   
10   SD Ke bawah    Jakartar Barat          15.0   
11   SD Ke bawah     Jakarta Utara          10.0   
12  DIV/S1/S2/S3  Kepulauan Seribu           1.0   
13  DIV/S1/S2/S3   Jakarta Selatan          12.0   
14  DIV/S1/S2/S3     Jakarta Timur          13.0   
15  DIV/S1/S2/S3     Jakarta Pusat           5.0   
16  DIV/S1/S2/S3    Jakartar Barat          16.0   
17  DIV/S1/S2/S3     Jakarta Utara          11.0   
18     DI/II/III  Kepulauan Seribu           1.0   
19     DI/II/III   Jakarta Selatan           5.0   
20     DI/II/III     Jakarta Timur           5.0   
21     DI/II/III     Jakarta Pusat           2.0   
22     DI/II/III    Jakartar Barat           7.0   
23     DI/II/III     Jakarta Utara           5.0   
24           SMK  Kepulauan Seribu           1.0   
25           SMK   Jakarta Selatan          39.0   
26           SMK     Jakarta Timur          41.0   
27           SMK     Jakarta Pusat          16.0   
28           SMK    Jakartar Barat          51.0   
29           SMK     Jakarta Utara          35.0   
30           SMA  Kepulauan Seribu           1.0   
31           SMA   Jakarta Selatan          31.0   
32           SMA     Jakarta Timur          32.0   
33           SMA     Jakarta Pusat          13.0   
34           SMA    Jakartar Barat          40.0   
35           SMA     Jakarta Utara          28.0   

    Total Kecamatan yang disampling  
0                               1.0  
1                              10.0  
2                              10.0  
3                               8.0  
4                               8.0  
5                               6.0  
6                               1.0  
7                              10.0  
8                              10.0  
9                               8.0  
10                              8.0  
11                              6.0  
12                              1.0  
13                             10.0  
14                             10.0  
15                              8.0  
16                              8.0  
17                              6.0  
18                              1.0  
19                             10.0  
20                             10.0  
21                              8.0  
22                              8.0  
23                              6.0  
24                              1.0  
25                             10.0  
26                             10.0  
27                              8.0  
28                              8.0  
29                              6.0  
30                              1.0  
31                             10.0  
32                             10.0  
33                              8.0  
34                              8.0  
35                              6.0

In [ ]:
#rancang_sampling['Sampel per Kecamatan'] = rancang_sampling['Total Sample'] / rancang_sampling['Total Kecamatan yang disampling']
#rancang_sampling

In [ ]:
rancang_sampling['Pembagian per Kecamatan'] = rancang_sampling.apply(
    lambda row: [row['Total Sample'] // row['Total Kecamatan yang disampling']] * int(row['Total Kecamatan yang disampling'])
                + ([row['Total Sample'] % row['Total Kecamatan yang disampling']] if row['Total Sample'] % row['Total Kecamatan yang disampling'] > 0 else []),
    axis=1
)

In [ ]:
# Mengonversi hasil ke integer
rancang_sampling['Pembagian per Kecamatan'] = rancang_sampling['Pembagian per Kecamatan'].apply(lambda lst: [int(x) for x in lst])

In [ ]:
rancang_sampling

Pendidikan              Kota  Total Sample  \
0            SMP  Kepulauan Seribu           1.0   
1            SMP   Jakarta Selatan          15.0   
2            SMP     Jakarta Timur          16.0   
3            SMP     Jakarta Pusat           6.0   
4            SMP    Jakartar Barat          20.0   
5            SMP     Jakarta Utara          14.0   
6    SD Ke bawah  Kepulauan Seribu           1.0   
7    SD Ke bawah   Jakarta Selatan          12.0   
8    SD Ke bawah     Jakarta Timur          12.0   
9    SD Ke bawah     Jakarta Pusat           5.0   
10   SD Ke bawah    Jakartar Barat          15.0   
11   SD Ke bawah     Jakarta Utara          10.0   
12  DIV/S1/S2/S3  Kepulauan Seribu           1.0   
13  DIV/S1/S2/S3   Jakarta Selatan          12.0   
14  DIV/S1/S2/S3     Jakarta Timur          13.0   
15  DIV/S1/S2/S3     Jakarta Pusat           5.0   
16  DIV/S1/S2/S3    Jakartar Barat          16.0   
17  DIV/S1/S2/S3     Jakarta Utara          11.0   
18     DI/II/III  Kepulauan Seribu           1.0   
19     DI/II/III   Jakarta Selatan           5.0   
20     DI/II/III     Jakarta Timur           5.0   
21     DI/II/III     Jakarta Pusat           2.0   
22     DI/II/III    Jakartar Barat           7.0   
23     DI/II/III     Jakarta Utara           5.0   
24           SMK  Kepulauan Seribu           1.0   
25           SMK   Jakarta Selatan          39.0   
26           SMK     Jakarta Timur          41.0   
27           SMK     Jakarta Pusat          16.0   
28           SMK    Jakartar Barat          51.0   
29           SMK     Jakarta Utara          35.0   
30           SMA  Kepulauan Seribu           1.0   
31           SMA   Jakarta Selatan          31.0   
32           SMA     Jakarta Timur          32.0   
33           SMA     Jakarta Pusat          13.0   
34           SMA    Jakartar Barat          40.0   
35           SMA     Jakarta Utara          28.0   

    Total Kecamatan yang disampling            Pembagian per Kecamatan  
0                               1.0                                [1]  
1                              10.0  [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 5]  
2                              10.0  [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 6]  
3                               8.0        [0, 0, 0, 0, 0, 0, 0, 0, 6]  
4                               8.0        [2, 2, 2, 2, 2, 2, 2, 2, 4]  
5                               6.0              [2, 2, 2, 2, 2, 2, 2]  
6                               1.0                                [1]  
7                              10.0  [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2]  
8                              10.0  [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2]  
9                               8.0        [0, 0, 0, 0, 0, 0, 0, 0, 5]  
10                              8.0        [1, 1, 1, 1, 1, 1, 1, 1, 7]  
11                              6.0              [1, 1, 1, 1, 1, 1, 4]  
12                              1.0                                [1]  
13                             10.0  [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2]  
14                             10.0  [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 3]  
15                              8.0        [0, 0, 0, 0, 0, 0, 0, 0, 5]  
16                              8.0           [2, 2, 2, 2, 2, 2, 2, 2]  
17                              6.0              [1, 1, 1, 1, 1, 1, 5]  
18                              1.0                                [1]  
19                             10.0  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5]  
20                             10.0  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5]  
21                              8.0        [0, 0, 0, 0, 0, 0, 0, 0, 2]  
22                              8.0        [0, 0, 0, 0, 0, 0, 0, 0, 7]  
23                              6.0              [0, 0, 0, 0, 0, 0, 5]  
24                              1.0                                [1]  
25                             10.0  [3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 9]  
26                             10.0  [4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1]  
27                              8.0  

The table above is the final output of the sampling design that will be implemented. Each district is plotted based on calculations of sample needs

### F.  Estimating the Needs of the Number of Samples

Metode yang digunakan untuk pengumpulan data adalah melalui telepon atau menggunakan layanan pesan daring. Metode pengumpulan data dengan sampling telepon merupakan salah satu cara untuk mengumpulkan data melalui telepon. Metode ini merupakan metode yang populer dalam melakukan survei dan riset pasar, karena relatif lebih murah dan cepat dibandingkan dengan metode lain seperti wawancara langsung atau penyebaran kuesioner.

Untuk menggunakan metode ini, Anda harus terlebih dahulu membuat daftar sampel yang akan dihubungi. Sampel ini dapat diambil secara acak dari populasi yang akan diteliti. Setelah itu, surveyor atau pewawancara akan menghubungi setiap nomor telepon yang ada dalam sampel dan mengajukan serangkaian pertanyaan yang diperlukan untuk mengumpulkan data.

Beberapa hal yang perlu diperhatikan dalam metode ini adalah:

1. Validitas Sampel: Pastikan bahwa sampel yang diambil benar-benar mewakili populasi yang ingin diteliti.
2. Keterampilan Pewawancara: Pewawancara harus memiliki keterampilan yang baik dalam mengelola wawancara telepon dan memastikan bahwa jawaban dari responden valid dan akurat.
3. Biaya: Meskipun relatif lebih murah dibandingkan dengan metode lain, biaya tetap dapat menjadi faktor yang signifikan tergantung pada jumlah sampel yang ingin diteliti.
4. Responden Tidak Tersedia: Beberapa responden mungkin tidak tersedia atau tidak bersedia berpartisipasi dalam wawancara telepon.
5. Keabsahan Jawaban: Jawaban yang diberikan oleh responden mungkin tidak akurat karena mereka mungkin tidak ingin terbuka atau merasa tidak nyaman dengan wawancara telepon.

### G.	Parameter Estimation Method

Berdasarkan hasil perancangan sampling yang telah dilakukan, dapat disimpulkan bahwa:

Dalam proses perancangan sampling, pengetahuan awal sangat diperlukan untuk memudahkan pemberian asumsi nilai pada parameter sampling.

Hasil perancangan sampling menunjukkan bahwa metode yang digunakan adalah multistage sampling, dengan stratifikasi pertama berdasarkan tingkat pendidikan, stratifikasi kedua berdasarkan kabupaten/kota, cluster sampling pada level kecamatan, dan simple random sampling pada level terendah.

Berdasarkan perhitungan perancangan sampling, diperlukan minimal 531 sampel untuk memperoleh gambaran statistik tentang tingkat pendapatan pekerja informal di DKI Jakarta, dan jumlah sampel tersebut dianggap memadai.

Biaya untuk melaksanakan sampling ini adalah sebesar Rp. 54.250.000, yang meliputi biaya tetap sebesar Rp. 4.000.000 dan biaya sampling untuk level PSU dan SSU sebesar Rp. 50.000 per sampel. Nilai ini masih tergolong terjangkau bagi instansi pemerintah untuk kebutuhan survei. Biaya ini masih merupakan asumsi "kasus terburuk," di mana biaya sebenarnya dapat dikurangi hingga 70-80%.

Untuk pengambilan sampel ini, metode telepon dan pesan daring digunakan untuk menanyakan kepada peserta survei mengenai jumlah pendapatan bulanan mereka. Dengan metode ini, penggunaan sumber daya dan alokasi surveyor menjadi lebih fleksibel.

Keuntungan dari desain sampling yang diterapkan meliputi:

* Stratifikasi sampling yang jelas memungkinkan
inferensi yang dapat dilakukan untuk masing-masing kabupaten/kota di DKI Jakarta.
* Metode multistage sampling lebih ekonomis dan efisien dibandingkan dengan SRS, mengingat populasi target yang besar dan tersebar secara geografis.
* Metode ini fleksibel dalam penggunaan berbagai teknik sampling.

Sedangkan kelemahan dari desain sampling ini adalah:

* Dibandingkan dengan SRS, metode multistage sampling memerlukan ukuran sampel yang lebih besar untuk mendapatkan inferensi statistik yang setara.
* Pemilihan metode sampling terbaik di setiap tahap bersifat subjektif, sehingga dibutuhkan alasan yang jelas untuk menghindari keputusan yang bias.
* Ada kemungkinan sampel yang dihasilkan tidak representatif karena sebagian besar populasi mungkin tidak terpilih, yang berpotensi menyebabkan bias undercoverage dan bias seleksi.

### J.	Conclusion


Untuk mengukur jumlah pengangguran di DKI Jakarta, diperlukan pengambilan sampel sebanyak 531 orang yang dikelompokkan berdasarkan tingkat pendidikan dan lokasi tempat tinggal.

Total biaya yang dikeluarkan untuk pengambilan sampel ini adalah Rp. 54.250.000, yang masih bisa dioptimalkan lebih lanjut.

Metode yang digunakan dalam pengambilan sampel ini adalah melalui telepon. Keuntungan dari metode ini adalah biaya yang lebih rendah dan efisiensi yang lebih tinggi untuk populasi yang tersebar secara geografis. Namun, kekurangan dari metode ini adalah kebutuhan akan ukuran sampel yang besar dan potensi bias akibat sampel yang mungkin tidak sepenuhnya representatif.










### K.	Reference

- https://www.healthknowledge.org.uk/public-health-textbook/research-methods/1a-epidemiology/methods-of-sampling-population

- https://jakarta.bps.go.id/id/publication/2021/10/28/445d7e1cdf4802df0fbdd8b8/profil-pengangguran-provinsi-dki-jakarta-2020.html